### PR TITLE
feat(frontend): add eval badge + alert

### PR DESCRIPTION
**Pull request by fatonramadani**
Screenshot 2023-05-03 at 14 56 46

### CODE REVIEW

File: frontend/src/lib/components/apps/editor/settingsPanel/script/shared/ScriptTriggers.svelte [hunk 1]
```diff
@@ -114,6 +116,12 @@
 					</span>
 				{/each}
 			</div>
+			{#if changeEvents.includes('Eval')}
+				<Alert type="info" title="Input evaluated at runtime" size="xs" class="mt-2">
+					At least one input is configured as an evaluated input. This means that the script will
+					run when the input value changes.
+				</Alert>
+			{/if}
```
Comment (rubenfiszel) [inline frontend/src/lib/components/apps/editor/settingsPanel/script/shared/ScriptTriggers.svelte]: would remove entirely

Comment (rubenfiszel) [reply frontend/src/lib/components/apps/editor/settingsPanel/script/shared/ScriptTriggers.svelte]: Instead, either adding a Tooltip in the eval badge, or a popover directly on the eval badge itself with the following text:
```
At least one input is configured as an evaluated input and the component will be triggered if the result of the eval change. Eval expressions are re-evaluated on any output or state changes.
```

Comment (fatonramadani) [reply frontend/src/lib/components/apps/editor/settingsPanel/script/shared/ScriptTriggers.svelte]: Done!

File: frontend/src/lib/components/apps/editor/settingsPanel/script/shared/ScriptTriggers.svelte [hunk 2]
```diff
@@ -96,6 +99,13 @@
 				{#each changeEvents as changeEvent}
 					<span class={classNames(badgeClass, colors['blue'])}>
 						{changeEvent}
+						{#if changeEvents.includes('Eval')}
+							<Tooltip class="!text-blue-600 ml-1">
```
Comment (rubenfiszel) [inline frontend/src/lib/components/apps/editor/settingsPanel/script/shared/ScriptTriggers.svelte:103]: you're adding the tooltip to every badge.

changeEvent == 'Eval'

**Pull request merged**